App.js

```diff
@@ -1,42 +1,70 @@
+import React, { useState, useEffect } from "react";
+import { StyleSheet,  View,  ScrollView } from 'react-native';
+import {
+  NavigationContainer,
+  DarkTheme as NavigationDarkTheme,
+  DefaultTheme as NavigationDefaultTheme
+   } from "@react-navigation/native";
+import {
+  Provider as PaperProvider,
+  DarkTheme as PaperDarkTheme,
+  DefaultTheme as PaperDefaultTheme
+ } from 'react-native-paper';
+import { createBottomTabNavigator } from "@react-navigation/bottom-tabs";
 import { Ionicons } from '@expo/vector-icons';
-import Friends from './components/friends.js';
-import ChatList from './components/chatlist.js';
-import CameraComponent from './components/camera.js';
-import Profile from './components/profileScreen.js';
-import Settings from './components/settingsScreen.js';
-import useToggle from './HelperFuncs/UseToggle.js';
-import EditProfile from './components/editProfileScreen.js';
-import LogoutScreen from './components/logoutScreen.js';
+import Friends from './components/friends.js'
+import ChatList from './components/chatlist.js'
+import CameraComponent from './components/camera.js'
+import Profile from './components/profile/profileScreen.js'
+import Settings from './components/profile/settingsScreen.js'
+import useToggle from "./HelperFuncs/profileHelpers.js";
+import EditProfile from "./components/profile/editProfileScreen.js";
+import LogoutScreen from "./components/profile/logoutScreen.js";
+import ChangePasswordScreen from "./components/profile/changePasswordScreen.js";
 import LoginPage from './components/loginPage';
 import SignupPage from './components/signupPage';
-import ChangePasswordScreen from './components/changePasswordScreen.js';
+
+import fakeUser from "./data/profileData.js";
 import data from './data/data';
 const Tab = createBottomTabNavigator();
 
 export default function App() {
+  // Add State that will be shared globally here
+  const [name, setName] = useState(fakeUser.first_name);
   const [profileSettingsOpen, setProfileSettingsOpen] = useToggle(false);
   const [editProfile, setEditProfile] = useToggle(false);
   const [logoutModalOpen, setLogoutModalOpen] = useToggle(false);
   const [changePassModalOpen, setChangePassModalOpen] = useToggle(false);
-  const [isLoggedIn, setLoggedIn] = useToggle(false);
-
-  const [email] = useState('Woofy@gmail.com');
+  const [isDarkTheme, setIsDarkTheme] = useToggle(false);
+  const [isLoggedIn, setLoggedIn] = useToggle(true);
+  const [email] = useState(fakeUser.email);
   const [userData, setUserData] = useState(data);
 
 
+  // Setting default and dark custom themes
+  const customDefaultTheme = {
+    ...NavigationDefaultTheme,
+    ...PaperDefaultTheme,
+    colors: {
+      ...NavigationDefaultTheme.colors,
+      ...PaperDefaultTheme.colors
+    }
+  };
+
+  const customDarkTheme = {
+    ...NavigationDarkTheme,
+    ...PaperDarkTheme,
+    colors: {
+      ...NavigationDarkTheme.colors,
+      ...PaperDarkTheme.colors
```

Comment:
I've never seen this syntax before, but I like how clean it is!

Reply:
Yea, the spread syntax is something I am still not totally comfortable with but trying to get more used to it